build.gradle.kts

```diff
@@ -14,11 +15,17 @@ dependencies {
   detektPlugins("io.gitlab.arturbosch.detekt:detekt-formatting:1.23.1")
 }
 
+allprojects {
+  version = "0.0.0"
```

Comment:
should it be 0.0.1 ?

Reply:
I'm still figuring out how this plays out with releases cut using GitHub. Might need to change after I test that out.